### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by jmlammert**

### CODE REVIEW

**Review by tkkuehn — COMMENTED** [review]
~This looks okay to me, but what was the problem? It looks to me like the output csv should be the same, but maybe I'm misunderstanding the pandas.~

Ah, I see it's addressing #2, sorry I had missed that on my first review.

File: hbn_postprocessing/html.py [hunk 1]
```diff
@@ -26,12 +26,16 @@ def check_html(
...
     out_path = Path(out_dir)
     matches.to_csv(out_path / "html-check_all.csv", sep=",")
-    matches[matches["html"] == "no"].to_csv(
+    no_html = matches[matches["html"] == "no"]
+    no_html.to_csv(
+    #matches[matches["html"] == "no"].to_csv(
         out_path / "html-check_no.csv",
         sep=",",
         index=False,
     )
```
Comment (tkkuehn) [inline hbn_postprocessing/html.py]:
```suggestion
    matches[matches["html"] == "no"].to_csv(
        out_path / "html-check_no.csv",
        sep=",",
    )
```

File: hbn_postprocessing/html.py [hunk 2]
```diff
@@ -26,12 +26,16 @@ def check_html(
...
-    matches[matches["html"] == "yes"].to_csv(
+    yes_html = matches[matches["html"] == "yes"]
+    yes_html.to_csv(
+    #matches[matches["html"] == "yes"].to_csv(
         out_path / "html-check_yes.csv",
         sep=",",
         index=False,
```
Comment (tkkuehn) [inline hbn_postprocessing/html.py]:
```suggestion
    matches[matches["html"] == "yes"].to_csv(
        out_path / "html-check_yes.csv",
        sep=",",
```

**Review by tkkuehn — CHANGES_REQUESTED** [review]
I think the cause of #2 is actually that I forgot to revert the `index=False` arguments in `to_csv` when I switched the index to be the `participant_id`, so I think what we need to do is get rid of those arguments, instead of naming the dataframe variables here.

I included a couple of suggestions that I think should do the job.

File: hbn_postprocessing/html.py (hunk 2 above)
Comment (jmlammert) [reply hbn_postprocessing/html.py]: Looks good, I will test with index=False removed.

**Review by tkkuehn — APPROVED** [review]

**Pull request merged**